packages/viem-utils/src/DeterministicDeployer/getTransaction.ts

```diff
@@ -92,6 +93,11 @@ export async function getOrPrepareDeterministicContract(
     };
 }
 
+export interface GetOrDeployDeterministicContractReturnType {
+    address: Address;
+    hash: Hash | undefined;
```

Comment:
hash?: Hash

Reply:
The key technically always exists so this is more accurate